pom.xml

```diff
@@ -974,6 +974,46 @@
             <linkXRef>true</linkXRef>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.basepom.maven</groupId>
+          <artifactId>duplicate-finder-maven-plugin</artifactId>
+          <version>1.5.1</version>
+          <configuration>
+            <printEqualFiles>false</printEqualFiles>
+            <failBuildInCaseOfDifferentContentConflict>false</failBuildInCaseOfDifferentContentConflict>
+            <failBuildInCaseOfEqualContentConflict>false</failBuildInCaseOfEqualContentConflict>
+            <failBuildInCaseOfConflict>false</failBuildInCaseOfConflict>
+            <checkCompileClasspath>true</checkCompileClasspath>
+            <checkRuntimeClasspath>true</checkRuntimeClasspath>
+            <checkTestClasspath>true</checkTestClasspath>
+            <skip>false</skip>
+            <quiet>false</quiet>
+            <preferLocal>true</preferLocal>
+            <useResultFile>true</useResultFile>
+            <resultFileMinClasspathCount>2</resultFileMinClasspathCount>
+            <resultFile>${project.build.directory}/duplicate-finder-result.xml</resultFile>
+            <!-- Version 1.1.1+ -->
```

Comment:
Do we need these version-specific comments? The specified version (1.5.1) seems to make these irrelevant.